modeltranslation/translator.py

```diff
@@ -157,7 +157,8 @@ def add_translation_fields(model, opts):
     model_empty_values = getattr(opts, 'empty_values', NONE)
     for field_name in opts.local_fields.keys():
         field_empty_value = parse_field(model_empty_values, field_name, NONE)
-        for lang in mt_settings.AVAILABLE_LANGUAGES:
+        available_languages = getattr(opts, 'allowed_languages', mt_settings.AVAILABLE_LANGUAGES)
```

Comment:
And, after we settled down on property name, let's move this to a property `translation_languages`. (or whaterer name you choose).